src/bindings.ts

```diff
@@ -66,16 +67,15 @@ export type UncheckedBindings = Partial<NonNullableFields<Bindings>>;
 export function checkMandatoryBindings(env: UncheckedBindings): Bindings {
 	if (env.ENVIRONMENT === undefined) throw new Error('ENVIRONMENT is undefined');
 	if (env.SERVICE === undefined) throw new Error('SERVICE is undefined');
-	if (env.DIRECTORY_CACHE_MAX_AGE_SECONDS === undefined)
-		throw new Error('DIRECTORY_CACHE_MAX_AGE_SECONDS is undefined');
 	if (env.ISSUANCE_KEYS === undefined) throw new Error('ISSUANCE_KEYS is undefined');
 	if (env.VERSION_METADATA === undefined) throw new Error('VERSION_METADATA is undefined');
 	if (env.KEY_LIFESPAN_IN_MS === undefined) throw new Error('KEY_LIFESPAN_IN_MS is undefined');
 	if (env.KEY_NOT_BEFORE_DELAY_IN_MS === undefined)
 		throw new Error('KEY_NOT_BEFORE_DELAY_IN_MS is undefined');
 
 	return {
-		DIRECTORY_CACHE_MAX_AGE_SECONDS: env.DIRECTORY_CACHE_MAX_AGE_SECONDS,
+		DIRECTORY_CACHE_MAX_AGE_SECONDS: env.DIRECTORY_CACHE_MAX_AGE_SECONDS ?? null,
+		USE_CACHE_API: env.USE_CACHE_API ?? null,
```

Comment:
may be worth having a default to `true`.

Reply:
nit: you could also part it so that you have a boolean instead of later checking `=== 'true'`